src/containers/ResultsPanel/selectors.ts

```diff
@@ -144,48 +141,27 @@ const configureData = (
     };
     // Check if the x-axis label has the word "time" in it, separated from other
     // characters by whitespace and/or one or more special characters
-    const isTimePlot = /\btime\b/i.test(xAxisTitle);
+    const isTimePlot = /\btime\b/i.test(plot.layout.xaxis.title);
 
-    // Add time indicator line for scatter plots with time on x-axis
-    if (isScatterPlot(inputData) && isTimePlot && currentTime !== 0) {
-        data.push({
-            /* cSpell:disable */
-            mode: "lines",
-            x: [currentTime, currentTime],
-            y: [0, 1],
-            xaxis: "x",
-            yaxis: "y2",
-            line: {
-                width: 1,
-                color: PLOT_STYLE.timeIndicatorColor,
-            },
-            showlegend: false,
-            hoverinfo: "x",
-            /* cSpell:enable */
-        });
-    }
-
-    return data;
+    return isScatterPlot(plot.data) && isTimePlot;
 };
 
 export const getPlotDataConfiguredForPlotly = createSelector(
-    [getPlotData, getCurrentTime],
-    (plotData: RawPlotParams[], currentTime: number): PlotParamsWithKey[] => {
+    [getPlotData],
```

Comment:
for consistently you could rename this to `getRawPlotParams` or something. (I know you didn't name this here)

Reply:
Oof, I looked into doing this but then it would be inconsistent with `state.metadata.plotData` and all of its references so I'll pass on this for now 😅